tests/Propel/Tests/Runtime/Map/DatabaseMapTest.php

```diff
@@ -101,27 +101,38 @@ public function testAddTableFromMapClass()
         }
     }
     
+    /**
+     * @return void
+     */
+    public function testRegisterTableMapClassByName()
+    {
+        $databaseMap = new DatabaseMapWithGetters('dummyDatabase');
+        $tableName = 'database.table_name';
+        $phpTableName = '\\php\\table\\name';
+        $tableMapClass = 'TableClass';
+        $databaseMap->registerTableMapClassByName($tableName, $phpTableName, $tableMapClass);
+        
+        $this->assertTableMapContains($databaseMap, $tableName, $phpTableName, $tableMapClass);
+    }
+    
     /**
      * @return void
      */
     public function testRegisterTableByMapClassAddsClassAsString()
     {
-        $databaseMap = new class('dummyDatabase') extends DatabaseMap {
-            public function getRawTables(){
-                return $this->tables;
-            }
-            
-            public function getRawTablesByPhpName(){
-                return $this->tablesByPhpName;
-            }
-        };
+        $databaseMap = new DatabaseMapWithGetters('dummyDatabase');
         
         $tableMapClass = BazTableMap::class;
         $databaseMap->registerTableMapClass($tableMapClass);
         
+        $this->assertTableMapContains($databaseMap, $tableMapClass::TABLE_NAME, '\\' . $tableMapClass::TABLE_PHP_NAME, $tableMapClass);
+    }
+
+    protected function assertTableMapContains(DatabaseMapWithGetters $databaseMap, $tableName, $phpTableName, $tableMapClass){
```

Comment:
```suggestion
    protected function assertTableMapContains(DatabaseMapWithGetters $databaseMap, $tableName, $phpTableName, $tableMapClass)
    {
```

I wonder if CS is not active for tests.

Reply:
Yes, tests are currently not linted